requirements.txt

```diff
@@ -1 +1 @@
-pyparsing==2.0.2

```

Comment:
I'd go for the latest version in hope they maintain backward compatibility. That would benefit you from stability and security standpoints.